test/javascript/packs/utils/calendar.test.js

```diff
@@ -1,4 +1,5 @@
-import { compareHours } from "../../../../app/javascript/packs/utils/calendar";
+import moment from 'moment';
+import { compareHours, prev, next, current, innerRange, startDate } from "../../../../app/javascript/packs/utils/calendar";
```

Comment:
`current` is not being used. Shouldn't we have a test for this function?

Reply:
Test added!